test/factories/scholarship_applications.rb

```diff
@@ -1,8 +1,7 @@
 FactoryGirl.define do
   factory :scholarship_application do
     reason 'MyText'
-    terms_accpeted false
-    references ''
-    references ''
+    terms_accepted false
+    user
```

Comment:
We will need this user to be a `verified: true` user.  Can you check out the FactoryGirl docs for the proper way to insert that here?